samples/meetings-notification/nodejs/.env

```diff
@@ -1,3 +1,3 @@
-MicrosoftAppId=
-MicrosoftAppPassword=
-BaseUrl=
\ No newline at end of file
+MicrosoftAppId=<<MICROSOFT_APP_ID>>
+MicrosoftAppPassword=<<MICROSOFT_APP_PASSWORD>>
+BaseUrl=<<BaseURL>>
```

Comment:
Kindly run build pipeline and share the link as well.

Reply:
Attach build pipeline link